src/components/ebay-dialog/index.marko

```diff
@@ -1,48 +1,24 @@
 import processHtmlAttributes from "../../common/html-attributes"
 
-static var ignoredAttributes = ["open", "type", "focus", "a11yCloseText"];
-
 $ var isFull = input.type === "full";
 $ var isCenter = !input.type || input.type === "fill";
 $ var isDocked = input.type === "left" || input.type === "right";
-<div
-    ...processHtmlAttributes(input, ignoredAttributes)
-    aria-modal="true"
-    role="dialog"
+<ebay-dialog-base
+    ...input
+    open=input.open
+    type="dialog"
+    on-modal-show('dialogShow')
```

Comment:
When re-emitting event's you don't need a `component.js` handler. You can simply (in this case):

```
on-modal-show("emit", "dialog-show")
on-modal-close("emit", "dialog-close")
```